addons/sale/migrations/11.0.1.1/post-migration.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_xmlid_renames = [
```

Comment:
@StefanRijnhart, @Benniphx I just want to ask, even if the PR is merged, if we have to do something with "_xmlid_renames" here?